src/out/admin/js/api-version-selector.js

```diff
@@ -0,0 +1,13 @@
+function selectMethod(selector)
+{
+    let selectedVersion = selector.value;
```

Comment:
i think `selectedVersion` could be a const here too. I understand the assumption but inside the function scope its value does not change